libs/barista-components/core/src/error/errors.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * @license
+ * Copyright 2020 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export function getDtCompareWithNonFunctionValueError(): Error {
+  return Error('`compareWith` must be a function.');
```

Comment:
Do we have to throw an error? Can we handle this more gracefully without crashing the page?. Maybe do nothing and use the dtLoggerFactory to log it to the dashboard.